models/news.js

```diff
@@ -22,6 +22,18 @@ exports.selectArticleById = (article_id) => {
     });
 };
 
+exports.selectCommentsByArticleId = (article_id) => {
+  return db
+    .query(
+      "select article_id, comment_id, votes, created_at, users.name AS author, body from comments INNER JOIN users ON comments.author = users.username where article_id = $1;",
+      [article_id]
```

Comment:
Firstly a stylistic point: it's fine to have your SQL commands be lower case, but I would recommend consistency. If your `select` is lower case but `JOIN` is upper, it's a little confusing to read!

Secondly, I'm not sure if we need to `JOIN` here. Because `author` is a foreign key, it must be referencing the `username` from the `users` table by default.

Reply:
Have made the SQL commands upper case.

Removed the join and now simply reference username from the articles table.